### ISSUE TITLE
Ver 2.0.2 for Windows does not work in Chrome/Jitsi

### ISSUE THREAD

**Issue by ghost** [CatxFish/obs-virtual-cam#34]
I am using the installer.  I follow the instructions for output.  I go to Chrome, go to settings, go to camera source and select OBS-Camera.  But when Using Jitsi, it insists on using my built-in camera for my laptop and says the camera can't be used (OBS is using it, but of course).  It refuses to use OBS.  I am thinking there's something wrong with Chrome or how OBS-virtual-cam interacts with Chrome?

![image](https://user-images.githubusercontent.com/43121564/45270016-ba97b080-b465-11e8-8ad5-c2459faef7e5.png)

![image](https://user-images.githubusercontent.com/43121564/45270033-f92d6b00-b465-11e8-912e-83e9b7c6007d.png)

EDIT: It works fine in Firefox.  But it's worth checking out in Chrome.

**Comment by CatxFish** [comment]
OK, I'll check later , there might be some violations.

**Comment by CatxFish** [comment]
I am not sure what is the reason , but after upgrade to chrome 69 , the virtual camera works fine with webRTC sample page.

**Comment by nineofcups49** [comment]
Yes, I see that too, after testing a sample here.  But jitsi refuses to work, only in Chrome.  [Try it.](https://meet.jit.si/advarksplaynicely).  The camera refuses to come up and use OBS-Camera and instead wants to circumvent OBS-Camera.  I will open an issue with Jitsi.

**Comment by ROBERT-MCDOWELL** [comment]
the problem is google chrome, not objs-virtual. so maybe you should contact them first

**Comment by CatxFish** [comment]
@nineofcups49 

Hi, I tried to use another device with built in webcam and find I don't know how to choose camera in jitsi, maybe jitsi has its own policy to use the first webcam. In the other way virtual cam works fine with google hangout. 

And I am sorry the chrome 69 solve the problem should be just a coincident when I do a rough test.

I think what you need is to ask jitsi how to choose another camera in their web interface so you can use virtual camera , or maybe find a way to disable the real webcam that the virtual camera will become the first choice.

**Comment by KovalevArtem** [comment]
After trying to switch from a real webcam to a obs-virtual-cam error, in the end, nothing works at all ...
![image](https://user-images.githubusercontent.com/36500228/46859813-82dea880-ce17-11e8-9025-332990f57d8a.png)

**Comment by KovalevArtem** [comment]
A [question ](https://github.com/CatxFish/obs-virtual-cam/issues/37)that is indirectly related to this topic.

**Comment by potatolordtwitch** [comment]
I had this problem as well, you need to right click the windows button and hit device manager, go into cameras and disable the camera chrome is trying to use, I might come up with a message that it will "stop it from functioning" but that just means you turned it off.

**Comment by Integrion** [comment]
Hi, does any one know why one can't use a virtual camera such as OBS or VMIX as the selected video camera device with Jitsi?  The virtual camera device shows up in the list of available devices, but selecting doesn't work. I've tried all the latest versions of the various browsers like Chrome, Firefox, Edge, and even Brave.  Anyone else running into the same issue?

**Comment by andredewerdt** [comment]
What worked for me was to stop OBS, then in firefox or chrome startup your jitsi meeting which will automatically use your webcam but now you have the option to switch to obs virtal cam which of course which will give still a blank screen because the obs isn't running. But then start OBS and start the virtual cam server. Everything is fine then.:

TLDR: Stop OBS, start Jitsi in browser, select virtual cam, start obs, start virtual cam server.
